installscripts/cookbooks/aws/recipes/default.rb

```diff
@@ -18,6 +18,5 @@
 end
 
 execute 'installaws-apigateway-importer' do
```

Comment:
we got rid of apigateway-importer in our v1.4.6 branch, perhaps this is a left over, can we remove it?